Tests/UnitTests/CMakeLists.txt

```diff
@@ -37,6 +37,20 @@ elseif(UNIX AND NOT ANDROID)
         Linux/App.cpp)
 endif()
 
+# Sanitizers
+option(ENABLE_SANITIZERS "Enable AddressSanitizer and UBSan" OFF)
+
+if(ENABLE_SANITIZERS)
+    set(ENABLE_RTTI ON CACHE BOOL "" FORCE)
+    if(CMAKE_CXX_COMPILER_ID MATCHES "Clang|GNU")
+        set(SANITIZERS "address,undefined")
```

Comment:
Note that it's vptr and fdsan that found the other issues so far. We can layer these in incrementally once we understand how much debt there is to clear out.